src/config.rs

```diff
@@ -22,6 +29,168 @@ impl fmt::Display for InstallError {
 
 impl std::error::Error for InstallError {}
 
+#[derive(Debug)]
+struct InstallStylesError;
+
+impl fmt::Display for InstallStylesError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        // XXX I feel this message is not very good – any idea for something better?
+        f.write_str("could not set the provided `Styles` globally as another was already set")
+    }
+}
+
+impl std::error::Error for InstallStylesError {}
+
+#[derive(Debug)]
+struct InstallColorBacktraceStylesError;
+
+impl fmt::Display for InstallColorBacktraceStylesError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        // XXX I feel this message is not very good – any idea for something better?
+        f.write_str("could not set the provided `Styles` via `color_backtrace::set_styles` globally as another was already set")
+    }
+}
+
+impl std::error::Error for InstallColorBacktraceStylesError {}
+
+// XXX now that we support text effects, "color scheme" doesn't seem accurate anymore. Let me know if you'd prefer a different name. The good thing about `Styles` is, that it's pretty short. "Theme" would be another good name, I just realized.
```

Comment:
I think Style is fine, though between `Style` and `Theme` I personally prefer `Theme`.

Reply:
You are right, I prefer "Theme" as well. I only realized "Theme" would be a good option when I made the last few changes, and I got a bit lazy here... 😅

I will rename "Styles" to "Theme" now.